src/classes/UTIL_OrgTelemetry_SObject_BATCH.cls

```diff
@@ -95,29 +102,57 @@ public class UTIL_OrgTelemetry_SObject_BATCH implements Database.Batchable<SObje
     }
 
     /**
-     * @description Execute the appropriate query to use in this batch job.
+     * @description Execute the appropriate query to use in this batch job. Clears out the last processed Id to prevent
+     * infinite loops
      */
     public Database.QueryLocator start(Database.BatchableContext context) {
+        String soqlQuery = buildQuery(lastProcessedId, chunkSize);
```

Comment:
@seanpat09 This was the part that I missed when testing the chaining with `this` concept in the Opp_Naming batch. The `start()` method has to recreate the query with the Id each time.  Works great once I made that change.